process/pipeline.go

```diff
@@ -92,42 +95,42 @@ If the input is an array, then an error is raised; the
 input should be run through the ForEach processor (which
 can encapsulate the Pipeline processor).
 */
-func (p Pipeline) Apply(ctx context.Context, cap config.Capsule) (config.Capsule, error) {
+func (p Pipeline) Apply(ctx context.Context, capsule config.Capsule) (config.Capsule, error) {
 	applicators, err := MakeApplicators(p.Options.Processors)
 	if err != nil {
-		return cap, fmt.Errorf("process pipeline: processors %+v: %v", p.Options.Processors, err)
+		return capsule, fmt.Errorf("process pipeline: processors %+v: %v", p.Options.Processors, err)
 	}
 
 	if p.InputKey != "" && p.OutputKey != "" {
-		result := cap.Get(p.InputKey)
+		result := capsule.Get(p.InputKey)
 		if result.IsArray() {
-			return cap, fmt.Errorf("process pipeline: inputkey %s: %v", p.InputKey, errPipelineArrayInput)
+			return capsule, fmt.Errorf("process pipeline: inputkey %s: %v", p.InputKey, errPipelineArrayInput)
 		}
 
 		newCap := config.NewCapsule()
```

Comment:
should this be `newCapsule` to match the `capsule` convention?

Reply:
Great point (and with `newCapsules` as well)! Modified these instances across the project with the latest fixup.